core/src/test/java/org/apache/iceberg/rest/TestRESTCatalog.java

```diff
@@ -737,6 +744,113 @@ public void testTableCredential() {
         ImmutableMap.of("Authorization", "Bearer client-credentials-token:sub=table-user"));
   }
 
+  @org.junit.Test
```

Comment:
Why the fully qualified name?

Reply:
That must have been automatically inserted, I'll remove it